src/main/resources/templates/db_scripts/schema.json

```
@@ -1253,6 +1253,11 @@
       "run": "after",
       "snippetPath": "inventory-hierarchy/updateRecordsViewFunctionToSupportLocationNameSeparately.sql",
       "fromModuleVersion": "28.1.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "inventory-hierarchy/fixElectronicAccessOrder.sql",
+      "fromModuleVersion": "29.0.1"
```

```suggestion
      "fromModuleVersion": "29.1.0"
```
Should it be this version?